lib/services/api/session_manager.ts

```diff
@@ -67,6 +67,7 @@ export class SNSessionManager extends PureService<SessionEvent> {
 
   private user?: User
   private isSessionRenewChallengePresented = false;
+  private currentSessionIsEphemeral = false;
```

Comment:
Eh, we have better ways of detecting this imo.

- https://github.com/standardnotes/snjs/blob/master/lib/services/storage_service.ts#L131
- Otherwise it should be added as a property of the session object itself.

Reply:
I ruled out adding ephemeral as a property on the session object because it made for something pretty messy, but the storage service method is exactly what I wanted, thanks :)